.changeset/proud-walls-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/console": patch
```

Comment:
Bump features module instead, since feature is the changed package in this PR

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/5970/commits/9a2b1db7c8dbada90d02332677a8e83bd5afee85